### ISSUE TITLE
Editing is unreasonably slow

### ISSUE THREAD

**Issue by aerkiaga** [calebmeyer/prismatic-parens#1]
For very large files, making any change renders the editor unresponsive for several minutes.

**Comment by calebmeyer** [comment]
Yeah, I need to eventually update this to take advantage of tree sitter. I haven't had much time recently. I'd be happy to accept PRs
